src/clts2vec/parse.py

```diff
@@ -103,23 +109,28 @@ def get_vec(self, sound, vectorize=True):
             >>> print(vec["cont"]
             - 1
         """
-        
-        sound = self.validate(sound)        
-        base_vec = Vector(self.binary_features) #{f: 0 for f in self.binary_features}
+
+        sound = self.validate(sound)
+        base_vec = Vector(self.binary_features)  # {f: 0 for f in self.binary_features}
 
         # check if sound is diphthong or complex
         complex_sound = False
+        sound_to_sound = ""
         if sound.endswith(" diphthong"):
+            self._apply_feature("diphthong", base_vec)
             sound_to_sound = sound[sound.index(" to ") + 4: -10] + " vowel"
             sound = sound[5: sound.index(" to ")] + " vowel"
+            # set up feature pairs for diphthong trajectory
+            diphthong_features = (["from_" + f for f in sound.split()[:-1]] +
```

Comment:
Looking at it again: why do you need "diphthong_features" here? They are not relevant in the remainder, or where are they used?

Reply:
sorry, just saw they are, forget my comment.